pkg/cli/mirror/operator.go

```diff
@@ -705,3 +717,25 @@ func validate(dc *declcfg.DeclarativeConfig) error {
 	}
 	return nil
 }
+
+// checkError will check validation errors
+// from operator registry and return a modified
+// error messages for mirror usage
+// FIXME(jpower432): Checking against the errors string could be an issue since
+// this is depending on the strings returned from `opm validate`. It would be better to propose that
+// the validation error are exposed here https://github.com/operator-framework/operator-registry/blob/master/alpha/model/error.go
+// and adds structure errors that return package and channels information.
+func checkErr(err error) error {
```

Comment:
@dinhxuanvu I am kind of going back and forth on whether adding this is worth it given it depends on string values from a library. An ideal situation would be if the `validationErr` type in operator registry was exposed and maybe defined as a structure error. Do you think that is possible in the future?